sync/sync-store/src/main/java/com/duckduckgo/sync/store/SyncStore.kt

```diff
@@ -37,6 +38,12 @@ constructor(
 
     private val isSignedInStateFlow = MutableStateFlow(isSignedIn())
 
+    init {
```

Comment:
do we need this `init` block? `isSignedIn` is already the default value of that flow

Reply:
the value was not emitted, but it’s not needed anymore since `RealSyncStateMonitor` is combining values. good catch.